Nim/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿Console.WriteLine("Welcome to Nim");
+Console.WriteLine("Rules are: You may only draw up to 3 matches.");
+
+int matches = 24; //Starting matches
+
+Console.WriteLine("|||||||||||||||||||||||| (24)"); //Presenting total matches
+Console.WriteLine("Player 1: How many matches do you want to draw?");
+
+round:
+tryAgain:
+
+int playerDraw = int.Parse(Console.ReadLine());
+
+if (playerDraw > matches)
+{
+    Console.WriteLine("You can't draw more than there is matches, try again!");
+    goto tryAgain;
+}
+
+if (matches == playerDraw) //Game Over if player loses
+{
+    Console.WriteLine("\nYou drew the last match. You lose!");
+    goto theEnd;
+}
+
+matches -= playerDraw; //Total matches left after players turn
+
+int matchCount = 0; //The printout
+displayMatches:
+matchCount++;
+if (matchCount <= matches)
+{
+    Console.Write("|");
+    goto displayMatches;
+}
+
+Console.Write($" ({matches})\n");
+
+//Ai's turn
+    
+int drawAI = matches % 4 == 0 ? 3 : matches % 2 != 0 ? 2 : 1; //With smarter AI; if ai is in certain position, it will try to put player in losing position (5, 9, 13, 17, 21)
```

Comment:
Pretty well done, but can be simplified, as discussed :D

Reply:
Sure, but how do I even simplify it even furthur? I cannot figure it out XD